assets/stylesheets/host.css

```diff
@@ -9,4 +10,229 @@ body {
 .frame {
   height: 100%;
   width:  100%;
+  margin-top: 15px;
+}
+
+.modal-container {
+  animation: modalContainerFade ease 0.5s;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 1;
+}
+
+@keyframes modalContainerFade {
+  0% {
+    background-color: rgba(0, 0, 0, 0);
+  }
+  100% {
+    background-color: rgba(0, 0, 0, 0.5);
+   }
+}
+
+.modal {
+  animation: modalFade ease 0.5s;
+  background-color: white;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 15px;
+  width: 1000px;
+  height: 550px;
```

Comment:
I know that I told you to statically size things, rather than worrying about percentages and stuff.  And I think that's working great on buttons and such.  However, I think it's a problem on this particular element.  When I pop it open, this modal is taking up more than my entire screen.  I think this should switch to being sized 90% (or so) in each dimension.

Reply:
Yep, I tried to resize this using percentages & hopefully, it looks better on your end now.